samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskNameGrouper.java

```diff
@@ -50,4 +51,8 @@
    * @return Set of containers, which contain the tasks that were passed in.
    */
   Set<ContainerModel> group(Set<TaskModel> tasks);
+
+  default Set<ContainerModel> group(Set<TaskModel> tasks, List<Integer> containersIds) {
+    return null;
```

Comment:
Returning null seems problematic to be used as the default behavior. What does it mean when caller invokes the grouper with a set of task models and a list of containerIds. Returning null implies that grouping cannot be done? Or that there was an error ??